models/classes/runner/session/TestSession.php

```diff
@@ -204,7 +204,7 @@ public function initItemTimer($timestamp = null)
         // try to close existing time range if any, in order to be sure the test will start or restart a new range.
         // if the range is already closed, a message will be added to the log
         $tags = $this->getItemTags($this->getCurrentRouteItem());
-        $this->getTimer()->start($tags, $timestamp)->save();
+        $this->getTimer()->end($tags, $timestamp)->save();
```

Comment:
don't we have tests for that? :thinking:

Reply:
For the timer, yes but not for this class. I can't find any test related to this method.